.golangci.yml

```diff
@@ -10,15 +15,30 @@ linters-settings:
     check-shadowing: true
   misspell:
     locale: US
+  maligned:
+    suggest-new: true
   
 linters:
   enable:
     - bodyclose
+    - dupl
+    - goconst
     - gocyclo
     - gofmt
     - goimports
     - golint
     - gosec
+    - maligned
     - misspell
+    - scopelint
     - unconvert
 
+issues:
+  exclude-rules:
+        # Exclude some linters from running on tests files.
```

Comment:
super nitpick. this line seems to be wrong indent

Reply:
Good catch, fixed